yaptide/celery/utils/manage_tasks.py

```diff
@@ -87,3 +92,54 @@ def translate_celery_state_naming(job_state: str) -> str:
         return EntityState.COMPLETED.value
     # Others are the same
     return job_state
+
+
+def handle_shieldhit_cancellation(tasks: list[CeleryTaskModel]):
+    """Function cancelling shieldhit processes"""
+    command_as_list = ['kill', '--signal', 'SIGINT']
```

Comment:
what about `shieldhit_kill_many_cmd` ?

Reply:
Task ids are appended to a command_as_list

Reply:
I just suggested variable rename, `command_as _list` is not so meaningful.